campaignion_layout/campaignion_layout.module

```diff
@@ -79,6 +79,11 @@ function campaignion_layout_page_build(&$page, $node = NULL) {
         $plugin->execute($layout['name']);
       }
     }
+    $page['#order_enabled'] = $layout['order_enabled'] ?? FALSE;
+    $page['#order'] = 'above-main';
```

Comment:
I do not care too much about the naming, I thought `above-main` is good enough. but sure, we can extend to `form-above-main` or use `above-content`, which might be more accurate. whichever you like most (I think a fresh look on the naming here is helpful indeed :innocent:)...

Reply:
I choose a string because they are easier extensible than a boolean, which might become a issue with this field. yes, if we are confident enough, that this can stay a boolean, OK with me.

Reply:
Naming: Yep, something that doesn’t suggest that the page is rendered above main (which doesn’t make sense) would be better.

For compressing this into one variable I was suggesting more something more like `$page['#order']` being either `FALSE` or maybe even `'default'` as long as `$layout['order_enabled']` is false. So only setting it to some value when it’s also enabled. This way the theme doesn’t have to implement the toggle logic.

Reply:
I would take up the suggestion from @katzmo to call it `reversed`.

Reply:
Ok, works as well but doesn’t have the advantage of being extensible. Fine for me though ;-)

Reply:
Implemented my own suggestion but left `#reversed` as-is.